### PR TITLE
Require Node.js 4 and ES2015ify

### PR DESCRIPTION
**Pull request by SamVerschueren**
I didn't ES2015ify yet. If you want, I can do it in this PR as well, just let me know.

Had to rewrite all the tests which used a lot of describes, so not sure about the entire refactoring. I merged the tests with the same titles and created a separate test file for the `globule` tests.

### CODE REVIEW

**Comment by sindresorhus** [comment]
> If you want, I can do it in this PR as well, just let me know.

Sure :)

**Comment by SamVerschueren** [comment]
ES2015ified as well

File: readme.md [hunk 1]
```diff
@@ -5,16 +5,15 @@
 
 ## Install
 
-```sh
-
+```
 $ npm install --save multimatch
```
Comment (sindresorhus) [inline readme.md]: We can remove the `--save` flag now because of npm5

**Comment by sindresorhus** [comment]
Thanks Sam :)

**Comment by SamVerschueren** [comment]
:beers:

File: readme.md (hunk 1 above)
Comment (SamVerschueren) [inline readme.md]: Good idea :).

**Pull request merged**